models/cow_protocol/ethereum/cow_protocol_ethereum_trades.sql

```diff
@@ -15,6 +15,8 @@ WITH
 trades_with_prices AS (
     SELECT evt_block_time            as block_time,
            evt_tx_hash               as tx_hash,
+           evt_index,
+           contract_address          as project_contract_address,
```

Comment:
If this was introduced only for the purpose of defining a Unique ID, we may not need it. Unless it is required by dex.trades table elsewhere.

I do, however agree that evt_index would ultimately ensure uniqueness of the key (in the event that a single partially fillable order is touched twice in the same transaction).

Reply:
> If this was introduced only for the purpose of defining a Unique ID, we may not need it. Unless it is required by dex.trades table elsewhere.
> 
> I do, however agree that evt_index would ultimately ensure uniqueness of the key (in the event that a single partially fillable order is touched twice in the same transaction).

As far as I'm aware it is part of the `dex.trades` model and that's why I've added it.

Maybe @soispoke can elaborate further if this is required or optional.

Reply:
Do we know yet about whether this should belong in a `dex.aggregators` table or if the `category` field will be added?